pymodm/compat.py

```diff
@@ -15,6 +15,11 @@
 """Tools for Python 2/3 compatibility."""
 import sys
 
+try:
+    import collections.abc as abc
+except ImportError:
+    # Python < 3.3
+    import collections as abc  # noqa
```

Comment:
What was the warning you needed to disable?

Reply:
F401 'collection as abc' imported but not used